documentation/upgrade/argo-checkout-0.8.0-to-0.9.0.md

```diff
@@ -4,7 +4,7 @@ In `@shopify/argo-checkout` and `@shopify/argo-checkout-react` version 0.9.x, si
 
```

Comment:
For both upgrade guides, I think you should leave all the package names exactly as they are (since the names of the packages need to line up with the versions we are referencing), but add a note at the top of the README that some packages discussed in this document have been renamed.